src/discordtipbot/discordbot/offsite.cr

```diff
@@ -0,0 +1,81 @@
+class Offsite
+  def initialize(@tip : TipBot, @config : Config)
+  end
+
+  def call(msg, ctx)
+    client = ctx[Discord::Client]
+    id = msg.author.id.to_u64
+
+    cmd_usage = String.build do |io|
```

Comment:
`String.build` is nice for very dynamic strings. This string does not change at all, but also, unlike the other commands, it doesn't actually print the command invocation (prefix / command name)

Best to just use a heredoc with interpolation:
```cr
cmd_usage = <<-DOC
  Usage: #{@config.prefix}offsite
  This command allows the storage of coins off site.

  - `address` Send coins here to deposit them again
  ...
  DOC
```